src/renderer/components/notifications/notifications.scss

```diff
@@ -3,10 +3,11 @@
 
   position: absolute;
   right: 0;
-  bottom: 0;
+  top: 0;
```

Comment:
It would be nice to consider keeping `bottom: 0` to locate the notification message box at the bottom-right corner. Which is a walk round of solving issue #842 as well. In that issue, the `draggable-top` div is outside the iframe which contains the notification box that make it difficult to find an ideal solution to have both text selection and drag features at the same time.
It is quite common to have notification on the bottom right as well.

Reply:
Notifications currently list top to bottom, so that placement wouldn't help much. I played around with not using `padding` (and trying to use `margin` instead) so that the visually empty space around the notifications can be clicked through. #842 needs other work not just this so I don't think that its work should bleed into this PR.